content/docs/languages/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Supported languages and platforms
```

Comment:
Should this just be "Supported languages"? I don't really see any platforms below.

Reply:
I guess you could argue "Android" and "Web" are platforms.

If we call out to languages and platforms distinctly, should we separate them? I.e. platforms in one list and languages in another?

Reply:
Thanks for your feedback @mjpitz.

> Should this just be "Supported languages"?

I juggled with that question too, but ...

> "Android" and "Web" are platforms.

Right. It doesn't make sense to qualify these as languages.

> If we call out to languages and platforms distinctly, should we separate them? I.e. platforms in one list and languages in another?

I'd vote to lease this as a unified list for now. (It is a small change that we can address later along with possible tweaking of the language. I'm eager to get this structural change in place first.)

Reply:
@mjpitz - if this is good enough as a step forward, and you feel that you questions you had are non-blocking, could you approve this PR?

Reply:
That should be fine. I didn't see any major issue with it. Was just odd because when I first looked, I didn't see any platforms. Needed to look again to find the web, etc.